docs/install/install-redis/_index.md

```diff
@@ -1,159 +1,159 @@
 ---
-title: "Install Redis"
-linkTitle: "Install Redis"
+title: "Install Valkey"
+linkTitle: "Install Valkey"
 weight: 1
 description: >
-    Install Redis on Linux, macOS, and Windows
+    Install Valkey on Linux, macOS, and Windows
 aliases:
 - /docs/getting-started/installation
 - /docs/getting-started/tutorial
 ---
 
-This is a an installation guide. You'll learn how to install, run, and experiment with the Redis server process.
+This is a an installation guide. You'll learn how to install, run, and experiment with the Valkey server process.
 
-While you can install Redis on any of the platforms listed below, you might also consider using Redis Cloud by creating a [free account](https://redis.com/try-free?utm_source=redisio&utm_medium=referral&utm_campaign=2023-09-try_free&utm_content=cu-redis_cloud_users).
+While you can install Valkey on any of the platforms listed below, you might also consider using Valkey Cloud by creating a [free account](https://redis.com/try-free?utm_source=redisio&utm_medium=referral&utm_campaign=2023-09-try_free&utm_content=cu-redis_cloud_users).
```

Comment:
We have no this kind of website so far, can we just remove this part?

Reply:
For this PR, I just add a minor comment here,

There are some links which include redis keyword, but i prefer updating them in a seperated pr as what @zuiderkwast said. Thanks